lib/paperclip/thumbnail.rb

```diff
@@ -76,7 +77,7 @@ def make
 
         parameters = parameters.flatten.compact.join(" ").strip.squeeze(" ")
 
-        success = convert(parameters, :source => "#{File.expand_path(src.path)}#{'[0]' unless animated?}", :dest => File.expand_path(dst.path))
+        success = convert(parameters, :source => "#{File.expand_path(src.path)}#{'[' + @frame_index.to_s + ']' unless animated?}", :dest => File.expand_path(dst.path))
```

Comment:
do you want me to put this on two lines?  didn't seem like the code was adhering to a max line count.  Also, Not sure the hash syntax here with having to wrap the frame_index in brackets. and only add this whole block based not he unless condition.  thoughts?